lib/api/conversions.js

```diff
@@ -125,10 +125,10 @@ module.exports = {
      * Change settlement date of a conversion
      * @param params
      * @params {String} params.id                 Id of the conversion (required)
-     * @params {Date} params.new_settlement_date  New settlement date (required)
+     * @params {Date} params.newSettlementDate    New settlement date (required)
      * @return {Promise}                          Promise; if fulfilled returns a json structure containing the details of the conversion date change; if rejected returns APIerror.
      * */
-    date_change: function (params) {
+    dateChange: function (params) {
```

Comment:
Hi @vrumger ,
First of all thank you for your interest to improve the SDK. 
The Snake Case style issue impacts more places. We have an internal task to deprecate these methods to have an aligned Camel Case consistent style which will be scheduled soon. I will close this PR in favour of the internal task.